src/main/java/com/commercetools/sync/products/utils/ProductVariantUpdateActionUtils.java

```diff
@@ -305,8 +306,8 @@
                 } catch (final BuildUpdateActionException buildUpdateActionException) {
                     final String errorMessage = format(FAILED_TO_BUILD_ATTRIBUTE_UPDATE_ACTION, attribute.getName(),
                         newProductVariant.getKey(), productKey, buildUpdateActionException.getMessage());
-                    syncOptions.applyErrorCallback(errorMessage,
-                        new BuildUpdateActionException(errorMessage, buildUpdateActionException));
+                    syncOptions.applyErrorCallback(new SyncException(errorMessage,
+                        new BuildUpdateActionException(errorMessage, buildUpdateActionException)), null, null, null);
```

Comment:
Should not we pass here old and new resource too? 
@heshamMassoud Did not we have a convention to call warningCallback on sync validation and errorCallback on API errors only?